src/modules/party/party.service.test.ts

```diff
@@ -225,7 +324,125 @@ describe('PartyService', () => {
     });
   });
 
-  function mockSuccessfulAcbsGetPartiesBySearchTextRequest(searchText: string, response: AcbsGetPartiesBySearchTextResponseElement[]): void {
+  describe('successful request using the getPartyIdentifierBySearchText method', () => {
+    it('returns the party identifier of the first matching party if the request is successful', async () => {
+      const searchText = 'searchText';
+
+      mockSuccessfulAcbsGetPartiesBySearchTextRequest(searchText, acbsGetPartiesResponse);
+
+      const response = await partyService.getPartyIdentifierBySearchText(idToken, searchText);
+
+      expect(response).toStrictEqual({
+        partyIdentifier: '00000000',
+      });
+    });
+
+    it('returns the party identifier of the first matching party if the query parameter searchText is exactly 3 characters and the request is successful', async () => {
+      const searchText = '999';
+
+      mockSuccessfulAcbsGetPartiesBySearchTextRequest(searchText, acbsGetPartiesResponse);
+
+      const response = await partyService.getPartyIdentifierBySearchText(idToken, searchText);
+
+      expect(response).toStrictEqual({
+        partyIdentifier: '00000000',
+      });
+    });
+
+    it('returns undefined if the request is successful and there are no matching parties', async () => {
+      const searchText = 'searchText';
+
+      // eslint-disable-next-line jest/unbound-method
```

Comment:
Please avoid `eslint-disable-next-line jest/unbound-method` can we follow previous PR approach please.